docgen/parser/parser.go

```diff
@@ -220,16 +220,35 @@ func parseMethods(pkgName string, messages map[string]*Message, methods []*googl
 			return nil, err
 		}
 
+		example, err := generateMethodExample(messages, m.GetOutputType())
+		if err != nil {
+			return nil, err
+		}
+
 		res[getQualifiedName(pkgName, m.GetName())] = &Method{
 			Name:      m.GetName(),
+			Request:   req,
 			Response:  messages[m.GetOutputType()],
+			Example:   example,
 			Operation: extOp.(*options.Operation),
-			Request:   req,
 		}
 	}
 	return res, nil
 }
 
+func generateMethodExample(messages map[string]*Message, name string) ([]string, error) {
+	_, ok := messages[name]
+	if !ok {
+		return nil, nil
+	}
+	p := genJSONExample(messages, name)
+	b, err := p.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+	return strings.Split(string(b), "\n"), nil
```

Comment:
forgot to ask earlier- might be missing something, but why the call to `strings.Split`? if example is just a string we can do

````
```json
{{$m.Example}}
```
````

Reply:
no idea why, removing.